launcher/container_runner.go

```diff
@@ -532,25 +540,53 @@ func initImage(ctx context.Context, cdClient *containerd.Client, launchSpec spec
 	return image, nil
 }
 
-func getImageLabels(ctx context.Context, image containerd.Image) (map[string]string, error) {
-	// TODO(jiankun): Switch to containerd's WithImageConfigLabels()
+// openPorts writes firewall rules to accept all traffic into that port and protocol using iptables.
+func openPorts(ports map[string]struct{}) error {
+	for k := range ports {
+		portAndProtocol := strings.Split(k, "/")
+		if len(portAndProtocol) != 2 {
+			return fmt.Errorf("failed to parse port and protocol, expect [port]/[protocol] like 80/tcp, got %s", portAndProtocol)
```

Comment:
nit: "failed to parse port and protocol: got %s, expected [port]/[protocol] 80/tcp"

Reply:
Done